### PR TITLE
add titles

### PR DESCRIPTION
**Pull request by turfle**

### CODE REVIEW

File: frontend/src/components/PageNotFound/index.tsx [hunk 1]
```diff
@@ -6,6 +7,10 @@ const PageNotFound = () => {
 
   return (
     <div className={classes.wrapper}>
+      <Helmet>
+        <title>{`${t('The page you were looking for doesn’t exist (404)')} - HypeCrafter`}</title>
+      </Helmet>
+
```
Comment (yevhenii-moroziuk) [inline frontend/src/components/PageNotFound/index.tsx]: Maybe we can create wrapper for it, eg:

`<Seo title={title} description={description} {...restProps} />`

Comment (turfle) [reply frontend/src/components/PageNotFound/index.tsx]: Yes we can, but I don't think that it is necessary. Maybe postpone it for later? Now we have tasks with much higher priority

Comment (yevhenii-moroziuk) [reply frontend/src/components/PageNotFound/index.tsx]: Yes, but we will never update this...

Comment (turfle) [reply frontend/src/components/PageNotFound/index.tsx]: Ok i'll take care of that

**Review by yevhenii-moroziuk — APPROVED** [review]

**Pull request merged**